src/interfaces.ts

```diff
@@ -506,6 +526,9 @@ export interface OnGetCodeActionsEvent<TFile extends BscFile = BscFile> {
     codeActions: CodeAction[];
 }
 
+export interface onPluginConfigureEvent {
```

Comment:
Small casing inconsistency:

```suggestion
export interface OnPluginConfigureEvent {
```